src/test/java/com/egov/core-services/eGovUser/passwordUpdate.feature

```diff
@@ -0,0 +1,65 @@
+Feature: Update user
+
+Background:
+    * def updatedUserPassword = read('../requestPayload/user/updatePassword/updatePassword.json')
+    * def userProfileData = read('../constants/user.yaml')
+    * def errorMessage = read('../constants/user.yaml')
+    * def existingPassword = userProfileData.validPasswordDetails.existingPassword
+    * def newPassword = userProfileData.validPasswordDetails.newPassword
+
+@Update_Password_ValidExistingPassword_validNewPassword_01 @positive @userPassword @user
+Scenario: To verify existing password is updating correctly
+        * call read('../pretests/updateUserPasswordPretest.feature@validPassword')
+        * assert updatedPasswordResponseBody.responseInfo.status == 200
+
+@Update_Password_ValidExistingPassword_InvalidNewPassword_02 @negative @userPassword @user
+Scenario: To verify the invalid length password error
+   * call read('../pretests/updateUserPasswordPretest.feature@inValidNewPassword')
+   * assert updatedPasswordResponseBody.Errors[0].code == errorMessage.errormessages.invalidPasswordLngthErrCode
+   * assert updatedPasswordResponseBody.Errors[0].message == errorMessage.errormessages.invalidPasswordLengthError
+   
+
```

Comment:
@krishnendu-eGov please remove extra empty lines from every file and keep it uniform as single line